src/DynamicRupture/Parameters.h

```diff
@@ -69,18 +69,20 @@ inline DRParameters readParametersFromYaml(YAML::Node& params) {
     initializers::updateIfExists(yamlDrParams, "rs_sr0", drParameters.rsSr0);
     initializers::updateIfExists(yamlDrParams, "rs_inisliprate1", drParameters.rsInitialSlipRate1);
     initializers::updateIfExists(yamlDrParams, "rs_inisliprate2", drParameters.rsInitialSlipRate2);
-    initializers::updateIfExists(yamlDrParams, "mu_w", drParameters.muW);
+    initializers::updateIfExists(yamlDrParams, "rs_muw", drParameters.muW);
 
     // Thermal Pressurization parameters
-    initializers::updateIfExists(yamlDrParams, "alpha_th", drParameters.thermalDiffusivity);
-    initializers::updateIfExists(yamlDrParams, "rho_c", drParameters.heatCapacity);
-    initializers::updateIfExists(yamlDrParams, "tp_lambda", drParameters.undrainedTPResponse);
-    initializers::updateIfExists(yamlDrParams, "initemp", drParameters.initialTemperature);
-    initializers::updateIfExists(yamlDrParams, "inipressure", drParameters.initialPressure);
+    initializers::updateIfExists(
+        yamlDrParams, "tp_thermaldiffusivity", drParameters.thermalDiffusivity);
+    initializers::updateIfExists(yamlDrParams, "tp_heatcapacity", drParameters.heatCapacity);
```

Comment:
tp_heatcapacity or TP_heatCapacity?
See consistency of other parameters in the same file.

Reply:
This is an (I think undocumented) feature of yaml-cpp: If you provide the string in all lower case, then it is case insensitive, if you provide at least one uppercase letter, it will be case sensitive.